### PR TITLE
feat: automated installer/zip on release tags

### PR DESCRIPTION
**Pull request by mrhoribu**
Automatically create an Inno Setup installer on a new release tag matching `v*.*.*` and add just core Lich-5 files to a zip/tarball for download as well.

### CODE REVIEW

**Comment by mrhoribu** [comment]
Implements and closes #199

**Comment by mrhoribu** [comment]
Also resolves #289

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ jinxp
 *.swp
 logo.png
 fly64.png
+fly64.ico
+R4LGTK3.iss
```
Comment (ondreian) [inline .gitignore:15]: this should include `scripts/` and `data/` now

**Review by ondreian — CHANGES_REQUESTED** [review]
almost there!

**Review by ondreian — APPROVED** [review]
lgtm

**Review by OSXLich-Doug — APPROVED** [review]
All in on this - remember to update the GUID with each major Ruby rev!

**Comment by ondreian** [comment]
what GUID? We can probably just make this match the git hash of the last
commit in the release, so it's got 1-to-1 relationship there?

**Comment by OSXLich-Doug** [comment]
Yeah, that might work.  GUID is in the innosetup script used to compile the
build.

**Comment by mrhoribu** [comment]
Hmm, need to figure out how to automate the new GUID creation and if it can be passed as a CLI option since the GUID is part of the config file. Wouldn't want to have to remember to update the GUID manually every time a new release is made.

> All in on this - remember to update the GUID with each major Ruby rev!

**Pull request merged**